nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/test/java/org/apache/nifi/processors/email/TestListenSMTP.java

```diff
@@ -44,7 +28,23 @@
 import org.apache.nifi.util.TestRunner;
 import org.apache.nifi.util.TestRunners;
 import org.junit.BeforeClass;
```

Comment:
This class is still referencing to JUnit 4 `BeforeClass` annotation, as opposed to the JUnit 5 `BeforeAll` annotation.  This is causing the test to fail with a NullPointerException due to the missing TLS configuration.